app/src/ea-timeline.ts

```diff
@@ -92,6 +92,22 @@ module ea {
             .attr('class', 'y axis')
             .attr('transform', 'translate(' + (-1 * spacing) + ',' + spacing + ')');
 
+
+        function resizePath(d) {
```

Comment:
should this function be assigned to a variable? So that handles can be defined by the lib user.